exercises/change/change_test.exs

```diff
@@ -30,6 +30,18 @@ defmodule ChangeTest do
   end
 
   @tag :pending
+  test "generates change with the least number of coins" do
```

Comment:
I ❤️ this change but I think the test names need to be more clear. Perhaps

"generates change using only small coins when it is not possible to combine them with larger coins" ???